docs/content/preview/yugabyte-voyager/migrate/live-migrate.md

```diff
@@ -527,6 +526,8 @@ Create a new database user, and assign the necessary user permissions.
    SELECT 'GRANT SELECT ON ALL SEQUENCES IN SCHEMA ' || schema_name || ' TO ybvoyager;' FROM information_schema.schemata; \gexec
    ```
 
+    Note that you may get "Permission Denied" errors for `pg_catalog tables` (such as pg_statistic). These errors do not affect the migration and can be ignored.
```

Comment:
@aishwarya24 can we get this added to the offline migration workflow as well. In offline workflow, there is no "postgres RDS" specifically, so you can add a note in the "postgres" tab itself, at the `GRANT SELECT ON ALL TABLES IN SCHEMA` as follows:

Note that **on RDS**, you may get "Permission Denied" errors for `pg_catalog tables` (such as pg_statistic). These errors do not affect the migration and can be ignored.